test/e2e/autoscaling_utils.go

```diff
@@ -204,7 +205,8 @@ func (rc *ResourceConsumer) CleanUp() {
 	expectNoError(rc.framework.Client.Experimental().HorizontalPodAutoscalers(rc.framework.Namespace.Name).Delete(rc.name, api.NewDeleteOptions(0)))
 }
 
-func runServiceAndRCForResourceConsumer(c *client.Client, ns, name string, replicas int) {
+// memLimit argument is in megabytes
```

Comment:
instead of writing this comment you can just rename the variable to sth like memLimitMb, cpuLimitMillis